addons/binding/org.openhab.binding.coolmasternet/src/main/java/org/openhab/binding/coolmasternet/internal/config/ControllerConfiguration.java

```diff
@@ -9,15 +9,21 @@
 package org.openhab.binding.coolmasternet.internal.config;
 
 /**
- * The {@link CoolMasterNetConfiguration} is responsible for holding
- * configuration information needed to access/poll the coolmasternet unit
+ * The {@link ControllerConfiguration} is responsible for holding configuration information needed to access/poll the
+ * CoolMasterNet Controller.
  *
- * @author Angus Gratton
+ * @author Angus Gratton - Initial contribution
+ * @author Wouter Born - Split Controller and HVAC configurations
  */
+public class ControllerConfiguration {
 
-public class CoolMasterNetConfiguration {
     public static final String HOST = "host";
+    public String host;
```

Comment:
I would prefer to put the constants at top, afaik its the general guideline

Reply:
Fixed!